tooltool.py

```diff
@@ -937,6 +947,11 @@ def main(argv, _skip_logging=False):
                            'files that cannot be distributed out of the company '
                            'but not for secrets; "public" files are available to '
                            'anyone withou trestriction')
+    parser.add_option('--ttl', default=None,
+                      type='int', dest='ttl', action='store',
+                      help='Time to live for the file to uploaded, in days; the '
+                           'files will be removed from storage, but the metadata '
+                           'will be retained')
```

Comment:
You may also want to mention that this is a "longest wins" TTL, so re-uploading the same file with a later TTL expiration will "extend" its life